gnomad_constraint/resources/resource_utils.py

```diff
@@ -92,6 +103,46 @@ def get_sites_resource(
         return public_release_grch38(data_type).versions[version]
 
 
+def get_coverage_ht(
+    data_type: str,
+    version: str = CURRENT_VERSION,
+) -> TableResource:
+    """
+    Return TableResource of coverage Table.
+
+    :param data_type: One of "exomes", "genomes" or "context.
+    :param version: One of the release versions (`VERSIONS`). Default is
+        `CURRENT_VERSION`.
+    :return: TableResource of coverage Table.
+    """
+    check_param_scope(version, data_type)
+    # TODO: decide path to coverage Table
+    return TableResource("")
+
+
+def get_annotated_context_ht(
+    version: str = CURRENT_VERSION,
+    use_old_data: bool = False,
```

Comment:
Maybe we shouldn't call this old data and make it more specific that this was for the v2 release, and add more details to the docstring about it